### PR TITLE
feat: http error handling without retries

### PR DESCRIPTION
**Pull request by vtsaplin**
## Description

What's inside:
- Removed the retry feature + related refactoring.
- Added error mapping for error code 429 (rate limit).
- Display a standard error message for unexpected issues.
- Respect the 60s execution limit and stop actions gracefully with a proper error message (408 status).

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: actions/AuthAction.js [hunk 1]
```diff
@@ -12,58 +12,58 @@
...
+        client_id: clientId,
+        type: 'access_token',
+      })
+      .headers({
+        Authorization: `Bearer ${token}`,
+        'Content-Type': 'application/json',
+      })
+      .get()
+      .json();
+    const { valid } = response;
```
Comment (basecode) [inline actions/AuthAction.js:32]: `return response.valid`?

**Review by basecode — APPROVED** [review]
LGTM

File: web-src/src/helpers/NetworkHelper.js [hunk 2]
```diff
@@ -10,12 +10,28 @@
...
+
+function wretchWithOptions(url) {
   return wretch(url)
-    .middlewares([retry({
-      retryOnNetworkError: true,
-      until: (response) => response && (response.ok || (response.status >= 400 && response.status < 500)),
-    })]);
+    .headers({ 'X-OW-EXTRA-LOGGING': 'on' })
+    .resolve((_) => {
+      return _
```
Comment (askayastha22) [inline web-src/src/helpers/NetworkHelper.js:26]: I would have preferred a parameter name such as `request`.

**Review by askayastha22 — CHANGES_REQUESTED** [review]
Left some minor comments.

**Review by askayastha22 — APPROVED** [review]

**Pull request merged**